doc/integrator/authentication.rst

```diff
@@ -42,3 +42,17 @@ To enable the two factors authentication you should set the following settings:
 
 If a user lost his second authentication factor he can't ask for a new one, to reset it the administrator
 should uncheck the 'The user changed his password' field on the user in the admin interface.
+
+Account lockout
+~~~~~~~~~~~~~~~
+
+To loc an account after a certain numer of failure set the following settings:
```

Comment:
To lock an account after a certain number of authentication failures, set the following: